internal/task/queue.go

```diff
@@ -74,8 +87,9 @@ func (t *Queue) Enqueue(obj interface{}) {
 		return
 	}
 	t.queue.Add(Element{
-		Key:       key,
-		Timestamp: ts,
+		Key:         key,
+		Timestamp:   ts,
+		IsSkippable: skippable,
```

Comment:
Can we piggyback on `Timestamp` field to implement "skippable" logic? For example if we want to enqueue an event that's not skippable we can set its Timestamp to current time + an hour. This way `t.lastSync > item.Timestamp` will not hold, and worker won't skip the event.

Reply:
done